### ISSUE TITLE
Add support for php 8.1

### ISSUE THREAD

**Issue by domeglic** [char0n/ffmpeg-php#63]
Doesn't work on php 8.1 because of classes implementing Serializable: https://php.watch/versions/8.1/serializable-deprecated

All those classes should:
- remove implementing interface serializable
- change serialize/unserialize methods into __serialize(): array and __unserialize(array $data).

```
Deprecated Functionality: Char0n\FFMpegPHP\OutputProviders\FFMpegProvider implements the Serializable interface, which is deprecated. 
Implement __serialize() and __unserialize() instead (or in addition, if support for old PHP versions is necessary) in 
/var/workspace/vendor/codescale/ffmpeg-php/src/OutputProviders/FFMpegProvider.php on line 7
```

**Comment by char0n** [comment]
Hi @domeglic,

I could successfully run the tests on PHP 8.1.9 and didn't even see the deprecation message - https://github.com/char0n/ffmpeg-php/runs/8110563318?check_suite_focus=true

What do you exactly mean by `Doesn't work on php 8.1`? It's a deprecation, it's not a breaking change right?

**Comment by domeglic** [comment]
I get an exception with the message I posted in the first post. But I am running this in developer mode on Magento2 so perhaps it is more strict.

**Comment by char0n** [comment]
@domeglic have a look at https://github.com/char0n/ffmpeg-php/pull/65

In my understanding, `Serializable` interface still can be used as long as we use `__serialize` and `__unserialize` magic methods. `Serializable` interface will be removed in PHP 9 which will be handled later.

**Comment by char0n** [comment]
Correction: I needed to remove `Serializable` interface, so that I don't need to implement it's methods.

**Comment by char0n** [comment]
Fixed in #65

**Comment by char0n** [comment]
Released as https://github.com/char0n/ffmpeg-php/releases/tag/3.2.2
